models/public/face-recognition-resnet34-arcface/accuracy-check.yml

```diff
@@ -11,14 +11,7 @@ models:
             shape: 3, 112, 112
 
     datasets:
-      - name: lfw
-
-        preprocessing:
-          - type: bgr_to_rgb
```

Comment:
Should we keep this preprocessing  for mxnet model?

Reply:
updated